src/core/main.cpp

```diff
@@ -820,8 +820,7 @@ int main( int argc, char * * argv )
 		if( autoSaveEnabled &&
 			gui->mainWindow()->getSession() != MainWindow::SessionState::Limited )
 		{
-			gui->mainWindow()->runAutoSave();
-			gui->mainWindow()->autoSaveTimerStart();
+			gui->mainWindow()->autoSave();
```

Comment:
Profiling? Never done it actually. I would expect Qt to bundle up timers together if it can and perform some threading if appropriate... ~ish.